bin/local-env/install-wordpress.sh

```diff
@@ -33,82 +30,96 @@ echo ''
 echo -e $(status_message "Ensuring that files can be uploaded...")
-docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm $CONTAINER mkdir -p /var/www/html/wp-content/uploads /var/www/html/wp-content/upgrade
-docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm $CONTAINER chmod 767 /var/www/html/wp-content/plugins /var/www/html/wp-config.php /var/www/html/wp-settings.php /var/www/html/wp-content/uploads /var/www/html/wp-content/upgrade
-
-CURRENT_WP_VERSION=$(docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run -T --rm $CLI core version)
+container mkdir -p \
+	/var/www/html/wp-content/uploads \
+	/var/www/html/wp-content/upgrade
+container chmod 767 \
+	/var/www/html/wp-content \
+	/var/www/html/wp-content/plugins \
+	/var/www/html/wp-config.php \
+	/var/www/html/wp-settings.php \
+	/var/www/html/wp-content/uploads \
+	/var/www/html/wp-content/upgrade
+
+CURRENT_WP_VERSION=$(wp core version)
 echo -e $(status_message "Current WordPress version: $CURRENT_WP_VERSION...")
 
 if [ "$WP_VERSION" == "latest" ]; then
 	# Check for WordPress updates, to make sure we're running the very latest version.
 	echo -e $(status_message "Updating WordPress to the latest version...")
-	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI core update --quiet
+	wp core update --quiet
 	echo -e $(status_message "Updating The WordPress Database...")
-	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI core update-db --quiet
+	wp core update-db --quiet
 fi
 
 if [[ ! -z "$GUTENBERG_VERSION" ]]; then
 	# Potentially install Gutenberg
 	echo -e $(status_message "Installing Gutenberg version $GUTENBERG_VERSION...")
-	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI plugin install gutenberg --version=${GUTENBERG_VERSION} --activate --force --quiet
+	wp plugin install gutenberg --version=${GUTENBERG_VERSION} --activate --force --quiet
 fi
 
 # If the 'wordpress' volume wasn't during the down/up earlier, but the post port has changed, we need to update it.
 echo -e $(status_message "Checking the site's url...")
-CURRENT_URL=$(docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run -T --rm $CLI option get siteurl)
+CURRENT_URL=$(wp option get siteurl)
 if [ "$CURRENT_URL" != "http://localhost:$HOST_PORT" ]; then
-	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI option update home "http://localhost:$HOST_PORT" --quiet
-	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI option update siteurl "http://localhost:$HOST_PORT" --quiet
+	wp option update home "http://localhost:$HOST_PORT" --quiet
+	wp option update siteurl "http://localhost:$HOST_PORT" --quiet
 fi
 
 # Install a dummy favicon to avoid 404 errors.
 echo -e $(status_message "Installing a dummy favicon...")
-docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm $CONTAINER touch /var/www/html/favicon.ico
+container touch /var/www/html/favicon.ico
+container chmod 755 /var/www/html/favicon.ico
```

Comment:
The lines above use `chmod 767`. Can we make this consistent or doesn't that work?
I know, super minor nitpick :-)